manifests/server/config.pp

```diff
@@ -139,7 +139,6 @@
     concat { $pg_ident_conf_path:
       owner  => $user,
       group  => $group,
-      force  => true, # do not crash if there is no pg_ident_rules
```

Comment:
Concat 2.0.x should continue to accept the force parameter, but throw a deprecation warning. If you remove the `force` param here, then any postgresql installation that is still running < 2.0.0, will fail... unless you also plan on changing the `metadata.json` to raise the minimum concat version to 2.0.0.